pkg/assert/service.go

```diff
@@ -0,0 +1,40 @@
+package assert
+
+import (
+	"net"
+	"strings"
+
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/gomega"
+)
+
+// CheckServiceSpecContainsSubstring retrives service specs and validates expected substring
+func CheckServiceSpecContainsSubstring(svc, expected string) {
```

Comment:
can we keep naming short and concise ?  idk, something like " CheckSvcSpec " ?  and then in the documentation you can do verbose if you want
// CheckSvcSpec retrives service specs and validates expected substring